recipes/tests/test_models.py

```diff
@@ -1,15 +1,23 @@
 from django.contrib.auth import get_user_model
 from django.test import TestCase
-from fixtureless.factory import create
+from datetime import timedelta
 
 from recipes.models import Recipe
 
 
 class TestRecipe(TestCase):
     def setUp(self):
-        super().setUp()
```

Comment:
you always want to super when subclassing a view. Django _could_ thoeretically include something in their `TestCase` class in the future that isn't backwards compatible and all your tests that didn't super would break. Extreme example, but always super :) sometimes where you super is important. In tests, it's usually best to super first. You can click up to the super call and see what's happening in the superclass though if you're not sure where to put the super

Reply:
Added back